modules/reactor/src/meshgenerators/PolygonConcentricCircleMeshGeneratorBase.C

```diff
@@ -928,5 +948,110 @@ PolygonConcentricCircleMeshGeneratorBase::generate()
   if (flat_side_up)
     MeshTools::Modification::rotate(*mesh0, 180.0 / (Real)_num_sides, 0.0, 0.0);
   mesh0->set_isnt_prepared();
+
+  if (_ring_block_ids.size() > 2 && getParam<bool>("replace_inner_ring_with_delaunay_mesh"))
```

Comment:
So the new feature is only applicable when you have more than two ring blocks?

(just some initial single comments. I will do a full review soon, thanks)

Reply:
I actually did not dig this much. I thought the first two elements of `_ring_block_ids` are for the inner ring because the inner ring may contain triangle and quad elements. This check is to insure that there is indeed a ring.